rhalphalib/model.py

```diff
@@ -290,7 +294,7 @@ def renderCard(self, outputFilename, workspaceName):
         observation = self.getObservation()
         if isinstance(observation, tuple):
             observation = observation[0]
-        signalSamples = [s for s in self if s.sampletype == Sample.SIGNAL]
+        signalSamples = [s for s in self if s.sampletype <= 0]
```

Comment:
Is sampletype not always just one of the enum values?

Reply:
I guess it's tricky, because what's signal and what isn't gets overwritten in `t2w` anyway, but for more signals they can be distinguish with negative numbers. `Sample.SIGNAL` is just `0`

Reply:
It should be automatically counting down from 0 for each kind of signal-like sample lower down in this function though. For sure this current interface doesn't really have much to say about multi-signal models but as you say anyway you can change everything during the text2workspace command

Reply:
I am not sure I follow. `Sample.SIGNAL` is always `0` no? I don't see it changed anywhere https://github.com/nsmith-/rhalphalib/blob/9adddbfce8a2ddc81f445e63716a1afdce9bf6a7/rhalphalib/sample.py#L20

Reply:
Signals get enumerated from negative numbers in the range in https://github.com/nsmith-/rhalphalib/blob/9adddbfce8a2ddc81f445e63716a1afdce9bf6a7/rhalphalib/model.py#L320

Reply:
Ah, I see. So this change literally does nothing. I was trying to use it when constructing the workspace to group signals by the number.

Reply:
Ah, I see. So this change literally does nothing. I was trying to use it when constructing the workspace to group signals by the number.

Reply:
In that case can it be reverted?